src/main/webapp/loginpage.xhtml

```diff
@@ -113,13 +113,9 @@
                                             <f:param value="#{systemConfig.guidesVersion}"/>
                                         </h:outputFormat>
                                         <h:outputFormat value="#{bundle['login.institution.support.blurbwithLink']}" escape="false">
-                                            <o:param>
-                                                <p:commandLink value="#{settingsWrapper.supportTeamName}" oncomplete="PF('contactForm').show()" update=":contactDialog" actionListener="#{sendFeedbackDialog.initUserInput}">
-                                                    <f:setPropertyActionListener target="#{sendFeedbackDialog.messageSubject}" value=""/>
-                                                    <f:setPropertyActionListener target="#{sendFeedbackDialog.recipient}" value="#{null}"/>
-                                                    <f:setPropertyActionListener target="#{sendFeedbackDialog.userMessage}" value=""/>
-                                                    <f:setPropertyActionListener target="#{sendFeedbackDialog.userEmail}" value=""/>
-                                                </p:commandLink>                               
+                                            <o:param>      
+                                                <p:commandLink value="Hello, #{settingsWrapper.supportTeamName}" onclick ="clickSupportLink();" >
```

Comment:
We don't want this English "Hello" here do we?

Reply:
Fixed...